cmd/scheduler.go

```diff
@@ -304,6 +304,12 @@ func run(options cli.OptionSet) {
 	tickChan := time.NewTicker(time.Second * 5).C
 	httpChan := make(chan string)
 
+	http.HandleFunc("/scheduler/ping", func(w http.ResponseWriter, r *http.Request) {
+		timestamp := time.Now().Unix()
+		logger.Infof("Pinging : %v", timestamp)
```

Comment:
Interesting, should I remove it from the other services as well? Or leave them until someone asks for them to be removed?